.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+dist
```

Comment:
please use empty line at the end of every file

Reply:
resolved